libtelemetry/src/test/java/com/mapbox/android/telemetry/TelemetryClientNavigationEventsTest.java

```diff
@@ -103,10 +108,20 @@ public GsonBuilder configure(GsonBuilder gsonBuilder) {
       }
     };
 
-  @Test
-  public void sendsTheCorrectBodyPostingAppUserTurnstileEvent() throws Exception {
+  @Before
```

Comment:
In general, we should avoid inline setup 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=65
Could you extract `setupMapboxTelemetry` code into a `private` "factory" method called for example `obtainMockedContext` and use it only in the tests that actually need it?
IMO, it reflects better what's going on in the test and you don't break the [triple A](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40) "rule".